src/symbolic_functions.cpp

```diff
@@ -25,82 +25,33 @@
 
 namespace amici {
 
-/**
```

Comment:
What's the reason for removing all those comments?

Reply:
They seem to have been moved to the corresponding header file. Which would make more sense to me, but I am not sure what the conventions are for C++ in-source documentation.

Reply:
Ah, sorry, missed that. Makes sense.